### PR TITLE
Remove commented code

### PR DESCRIPTION
**Pull request by Ozaq**
There was a lot of commented code that made some files very noisy.

Commented code now removed and in some places superflous coments have
been removed as well.

Fixes: #505

### CODE REVIEW

File: jpscore/src/main.cpp [hunk 1]
```diff
@@ -74,41 +73,28 @@ int main(int argc, char ** argv)
...
     if(sim.InitArgs()) {
-        // evacuation time
         double evacTime = 0;
         LOG_INFO("Simulation started with {} pedestrians", sim.GetPedsNumber());
         if(sim.GetAgentSrcManager().GetMaxAgentNumber()) {
-            // Start the thread for managing the sources of agents if any
-            // std::thread t1(sim.GetAgentSrcManager());
             double simMaxTime = config.GetTmax();
             std::thread t1(&AgentsSourcesManager::Run, &sim.GetAgentSrcManager());
-            while(!sim.GetAgentSrcManager().IsRunning()) {
```
Comment (chraibi) [inline jpscore/src/main.cpp]: Any reason why you removed this while?

Comment (Ozaq) [reply jpscore/src/main.cpp]: Hi @chraibi good catch /  question! :1st_place_medal: 

I did remove it because "Hey the loop is not doing anything expect spinning w.o. purpose, i.e. no loop body". 

However:
Your review remark promoted me to look into the code and be sure there are no side effects: Turns out there are! :/ The loop is actually blocking the main thread until `AgentSrcManager` has entered its `Run` method. Now the thing is: The loop is probably still useless because although it blocks the main thread until Run has been entered no additional work (that I can see) has been completed before continuing in the main loop. If the intention was here to sync execution of the `AgentSrcManager` with the main sim loop then it is questionable if this works as intended.

In conclusion, I would still remove this loop.

Comment (chraibi) [reply jpscore/src/main.cpp]: Maybe this leads us to another issue, to be opened soon, about "How to force both threads to run nearly at sync?"

Comment (Ozaq) [reply jpscore/src/main.cpp]: Actually Id much prefer to just remove the thread completely: See #628

File: libcore/src/Simulation.h [hunk 2]
```diff
@@ -198,13 +185,9 @@ class Simulation
     bool correctGeometry(std::shared_ptr<Building> building, std::shared_ptr<TrainTimeTable>);
 
     /**
-     * Updates the output filename if the current file exceeds 10MB.
+     * Updates the output filename if the current file exceeds 16MB.
```
Comment (zeroset) [inline libcore/src/Simulation.h]: I would prefer mentioning `_maxFileSize` here, instead of 16 MB.

Comment (Ozaq) [reply libcore/src/Simulation.h]: Fixed
